src/fluree/server/consensus/standalone.clj

```diff
@@ -36,10 +30,9 @@
                             fluree/db
                             (fluree/stage txn opts)
                             deref!)
-          commit-result (deref!
-                         ;; following uses :file-data? and will return map with {:keys [db data-file commit-file]}
-                         (fluree/commit! ledger staged-db {:file-data? true}))
-          broadcast-msg (events/transaction-committed params commit-result)]
+          result        (deref!
+                         (fluree/apply-stage! ledger staged-db))
```

Comment:
What is `apply-stage` do here? When I try to load it my repl it says no definitions found.

Reply:
Ok, I found the branch, I'm not sure why `xref` wasn't working for me. So it looks like `apply-stage` is just commit with a wrapper.

I'm kind of ambivalent about having this as a top-level api - we call directly into the `connection` ns in other places, could we do that here too instead of making `apply-stage` top-level?

Reply:
This used to be a top level api because it was part of the `commit!` function. You had to pass the `:file-data? true` option to get the output to have this shape. I don't think we should be doing this kind of control flow with options, so I split that functionality up into two separate functions. I made it a top level api because that functionality used to be a top level api, and I can see other users wanting to know the commit address after calling the commit.